queue/index.d.ts

```diff
@@ -0,0 +1,34 @@
+import { Action, Meta } from '@logux/core'
```

Comment:
It will not be a public API, we can remove types

Reply:
Ok, hopefully they were at least a bit helpful, when looking at this pr :smile: